main.rb

```diff
@@ -9,6 +10,20 @@ def initialize
 
   def prompt
```

Comment:
- [ ] It would be better if we display the stored data on start up, currently we are not displaying the data on start up
![Screenshot from 2023-08-03 07-11-53](https://github.com/Bifabig/school-library-ruby/assets/46388113/fcf674dc-018e-409d-9ba7-5165854791fe)